frontend/app/src/store/balances/getters.ts

```diff
@@ -278,6 +279,10 @@ export const getters: GetterTree<BalanceState, RotkehlchenState> = {
     }, new Array<AccountDSRMovement>());
   },
 
+  dsrHistoryByAddress: ({ dsrHistory }: BalanceState): DSRHistory => {
+    return dsrHistory;
```

Comment:
this doesn't seem to do anything other than providing the dsrHistory from the state.

Reply:
The reason I did this is because a `dsrHistory` already exists but I couldn't seem to get the info we needed out of it. Then I couldn't figure out how to get dsrHistory just from the state without the name conflicting with the `dsrHistory` balance getting which is also used in the same component (Lending). Ideas?

Reply:
mapState/mapGetter getters usually perform some trasformation if you just need the history you have to mapState instead

Reply:
you can also alias the mapping if they overlap https://vuex.vuejs.org/guide/state.html#the-mapstate-helper

Reply:
So what happens when we have to do (as in Lending)
```
computed: {
    ...mapGetters(['dsrHistory']),
    ...mapState(['dsrHistory'])
}
```

How do we access both?

Reply:
So I can do like this?
```
computed: {
    ...mapGetters(['dsrHistory']),
    ...mapState({dsrHistoryByAddress: 'dsrHistory'})
}
```

Reply:
Exactly.